java/google/registry/rde/imports/RdeHostLinkAction.java

```diff
@@ -117,10 +117,18 @@ public void map(JaxbFragment<XjcRdeHostElement> fragment) {
         ofy().transact(new VoidWork() {
           @Override
           public void vrun() {
+            // link host to superordinate domain and set time of last superordinate change to
+            // the time of the import
             HostResource host =
                 ofy().load().now(Key.create(HostResource.class, xjcHost.getRoid()));
```

Comment:
also if I'm incrementing a counter inside the transaction, it's not going to be accurate.